redhat-director-scripts/common/trilio.repo

```diff
@@ -0,0 +1,5 @@
+[triliovault-4-1]
+name=triliovault-4-1
+baseurl=http://trilio:XpmkpMFviqSe@repos.trilio.io:8283/triliovault-4.1-dev/yum/
```

Comment:
you may consider the user/password as params to the script and doing something like...
cat > trilio.repo <<EOF
[triliovault-4.1]
name=triliovault-4-1
baseutl=http://${USER}:${PASS}@repos...
...
EOF

Reply:
Yes, we need implement this.

Reply:
Implmented.